modules/openxr/openxr_interface.cpp

```diff
@@ -918,14 +918,29 @@ void OpenXRInterface::process() {
 				handle_tracker(trackers[i]);
 			}
 		}
+
+		// Handle hand tracking
+		OpenXRHandTrackingExtension *hand_tracking_ext = OpenXRHandTrackingExtension::get_singleton();
+		if (hand_tracking_ext && hand_tracking_ext->get_active()) {
+			for (int i = 0; i < MAX_OPENXR_TRACKED_HANDS; i++) {
+				OpenXRInterface::Tracker *tracker = find_tracker(i == 0 ? "/user/hand/left" : "/user/hand/right");
```

Comment:
This looks fragile. A better approach would be to move the logic below into a function that takes in the XrPath:
```
handle_hand_tracking("/user/hand/left");
handle_hand_tracking("/user/hand/right");
```

Reply:
We could, but it'll still remain index based as it's hardcoded in the spec that we just access the two hands in hand tracking.

It is us in this case that map hand tracking to the xrPaths which is something I never got why when the hand tracking API was added to OpenXR, wasn't how it worked in the first place.

I'm guessing because while the OpenXR spec focussed on the left hand and right hand, and any controller held would be bound, to a lot of XR runtime implementations treating `/user/hand/left` not as the left hand, but as the left controller. This likely because it was the approach common before OpenXR changed it. This has led to a lot of issues that were only corrected since the 1.0.28 OpenXR spec, and still has a few left over edge cases in certain XR runtimes.

Reply:
Ok, let me know what you think of how I changed it.